README.md

```
@@ -1,96 +1,155 @@
+---
+![App Architecture Diagram](images/login.png) 
+---
+**Project Features**
+-------------------
+
+* Jenkins pipeline for building, testing, and deploying a software project
+* EC2 instance as a master node and EC2 instance as a worker node
+* Dockerfile for building the project image
+* Jenkinsfile for configuring the Jenkins pipeline
+
+**Pre-requisites**
```

Markdownlint (0.35.0)

Expected: atx; Actual: setext
Heading style

(MD003, heading-style)